src/core/load-fragment.c

```diff
@@ -5199,6 +5199,34 @@ int config_parse_temporary_filesystems(
         }
 }
 
+int config_parse_private_tmp(
+                const char* unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        ExecContext *c = ASSERT_PTR(data);
+        int r;
+
+        assert(filename);
+        assert(rvalue);
+
+        r = parse_boolean(rvalue);
+        if (r < 0) {
+                log_syntax(unit, LOG_ERR, filename, line, r, "Failed to parse boolean value: %s ignoring", rvalue);
+                return -ENOEXEC;
```

Comment:
You are not actually ignoring the failure here

Reply:
not resolved.

Reply:
i.e. i think the error msg was right, you should ignore this, but you didn't

Reply:
I thought you meant the other way around, fixed